pkg/plugins/ifaces.go

```diff
@@ -56,6 +56,7 @@ type FS interface {
 
 	Base() string
 	Files() []string
+	Remove() error
```

Comment:
Nit: Personally I would probably change this to be a separate interface, and checking if it is implemented before calling Remove(). This way it should be easier to handle FS implementations that do not need to remove files (like CDN)

```go
type FSRemover interface {
	Remove() error
}
```

and in `loader.unload`:

```go
	if remover, ok := p.FS.(plugins.FSRemover); ok {
		if err := remover.Remove(); err != nil {
			return err
		}
	}
```